cmdstanpy/model.py

```diff
@@ -474,9 +474,14 @@ def compile(
                     self._compiler_options.add(compiler_options)
         exe_target = os.path.splitext(self._stan_file)[0] + EXTENSION
         if os.path.exists(exe_target):
-            src_time = os.path.getmtime(self._stan_file)
             exe_time = os.path.getmtime(exe_target)
-            if exe_time > src_time and not force:
+            included_files = [self._stan_file]
+            included_files.extend(self.src_info().get('included_files', []))
+            out_of_date = any(
+                os.path.getmtime(included_file) > exe_time
```

Comment:
How does `os.path.getmtime` work if a file doesn't exist? It is unlikely, but it's possible that a file gets deleted between line 479 and 481 here, so it would be good to check if it exists if `getmtime` fails badly.

Reply:
The function raises `FileNotFoundError` if the file doesn't exist. I imagine that's a reasonable error to raise here without wrapping in a custom exception?

Reply:
Fair enough!